### DISCUSSION TITLE
Welcome to TRUDUtilsD365 Discussions!

### DISCUSSION THREAD

**Discussion by TrudAX** [TrudAX/TRUDUtilsD365#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bellasommer** [comment]
Does the tool need to be updated in order to work with 2019?

**Reply by TrudAX** [reply]
Hi @bellasommer . I didn't test it, it probably depends on libraries used by Microsoft. I will do an upgrade as soon as Microsoft releases a downloadable one-box with VS2019

**Reply by bellasommer** [reply]
Hi @TrudAX  This is why I ask, I just deployed the new 10.0.21 version one box, and it is coming with 2019
/ Isabella
